### PR TITLE
Create Data_in_life_blog.com.json

### PR DESCRIPTION
**Pull request by jhylin**
Adding my blog, which uses R in different data projects, to the list.

### CODE REVIEW

**Comment by jhylin** [comment]
Hi, I'm not very experienced with doing a pull request, but did try to follow the instructions provided in the contributing.md file. I have a feeling perhaps it's due to using the GitHub Pages web URL. If so, I do have my blog also hosted on Netlify if that helps (https://jhylin-portfolio.netlify.app/). The other thing is that I don't seem to come across a place to add @Athanasiamo according to the contributing.md file, so my apologies if I missed it. Thank you.

**Comment by cosimameyer** [comment]
@jhylin thanks so much for submitting it! A quick question: Did you add the category (R or R-post for instance) to you RSS feed? I believe you have a distill blog, so it should be similar to the issue raised [here](https://github.com/rstudio/distill/issues/486). I'm not sure if it will work (please let me know if it doesn't and we can add it to the issue). Having it in general would be fantastic to keep the bots' feed sorted ☺️

**Comment by jhylin** [comment]
@cosimameyer Thank you for inviting me to add my blog here! I've added category into my RSS feed, and commited this the 2nd time, the errors were reduced from two to one, then I realised that the file name I used previously might not be correct, it probably should be jhylin.github.io.json only (rather than jhylin.github.io/Data_in_life_blog.com.json). I've made this change and pushed to the patch-1 branch on my forked repository jhylin/awesome-rladies-blogs, and not sure what to do next?

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Thank you so much for submitting.

You have placed you entry in a subfolder og blogs, which we dont accept. Could you pleave move your file

`blogs/jhylin.github.io/Data_in_life_blog.com.json` 

to 

`blogs/jhylin.github.io.json` 


thanks!

**Comment by cosimameyer** [comment]
And thanks so much for the changes in your RSS feed, @jhylin! It looks like it still ends with your_domain/index.xml while it should be something like this: https://cosimameyer.com/category/r-post/index.xml (that's how it looks like for my feed - you can also see it [here](https://simplepie.org/demo/?feed=https%3A%2F%2Fcosimameyer.com%2Fcategory%2Fr-post%2Findex.xml)). I'm wondering why it's not working properly for distill blogs but we can add your example also as a reference to the issue to help figure out why it is the case 😊

**Comment by karbartolome** [comment]
> And thanks so much for the changes in your RSS feed, @jhylin! It looks like it still ends with your_domain/index.xml while it should be something like this: https://cosimameyer.com/category/r-post/index.xml (that's how it looks like for my feed - you can also see it [here](https://simplepie.org/demo/?feed=https%3A%2F%2Fcosimameyer.com%2Fcategory%2Fr-post%2Findex.xml)). I'm wondering why it's not working properly for distill blogs but we can add your example also as a reference to the [issue](https://github.com/rstudio/distill/issues/486) to help figure out why it is the case 😊

Hi! Not sure if this helps, but regarding the RSS feed for a specific category, I've found that in distill blogs the categories folder is generated when a post is rendered. However, this folder is eliminated if the blog is rendered. A categories folder, containing each specified category with an index.xml for each category is what should be included in the site in order to get a category based feed. I've opened an [issue ](https://github.com/rstudio/distill/issues/486) for this in the distill repo.

**Comment by jhylin** [comment]
@drmowinckels - Thank you, I did manage to figure out how to move the file to blogs/jhylin.github.io.json from my forked repository (hopefully that was done properly, please let me know if it was not).

@cosimameyer and @karbartolome - Thank you for all of your messages, sorry I forgot to mention that I've built my blog from Quarto, does it also have the same RSS feed issue mentioned for the Distill blogs? 

After I've made another commit again today, the error message was now different from before, and I'm not entirely sure where the error came from, perhaps the RSS issue then? (I followed the instructions provided from the Quarto reference on how to add RSS feeds into the blog - the section on "Category Feeds" - https://quarto.org/docs/websites/website-blog.html#rss-feed).

**Comment by cosimameyer** [comment]
@jhylin, if you're not using distill (that's my bad, I'm losing track of all the different blogs 😄), [this answer](https://community.rstudio.com/t/setting-up-rss-feed-for-a-category-in-quarto-blog/146489/3) could be helpful ☺️

**Comment by cosimameyer** [comment]
Thanks so much! It looks fantastic  ❤️ 🤩

Just thinking out loud but it might be helpful to add the solutions/approaches as guides to the contribution guide as well (I'll take care of it once I'm back at a computer and create a suggestion in a PR ☺️) - which solution did work for you, @jhylin? ☺️

**Comment by jhylin** [comment]
@cosimameyer - Oh it worked! I thought it did not... Here's what I've done:
1. For people using GitHub Pages to deploy Quarto blogs, end the filename as "...username.github.io.json", without the 2nd part with the repository name for the blog
2. For RSS feeds in Quarto blogs: changed the code in index.qmd as (under listing):
feed: 
    categories: [R]
*Note to new users that the category names will be the names of your category tags used in the blogs (not "posts", which are the posts folder for Quarto blogs)
*Then provide the RSS feed links as, "https://username.github.io/blog_repo_name/index-r.xml" for R category posts ("https://username.github.io/blog_repo_name/index.xml" will be the RSS feed link for main posts only)
3. Remove any spaces in file names, and consider replacing it with underscores or other options.

Hope by adding above into the contribution.md will clarify some of the instructions for new users.
Many thanks!

File: blogs/jhylin.github.io.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Data in life",
+  "type": "blog",
+  "url": "https://jhylin.github.io/Data_in_life_blog/",
+  "photo_url": "https://github.com/jhylin/Data_in_life_blog/blob/main/profile_avatar.jpg",
+  "description": "A blog using data science on pharmaceutical and healthcare data",
+  "language": "en", 
+  "rss_feed": "https://jhylin.github.io/Data_in_life_blog/index-r.xml"
```
Comment (cosimameyer) [inline blogs/jhylin.github.io.json:8]: Here should be a comma after your feed entry ☺️

**Comment by cosimameyer** [comment]
Thank you! That's really helpful ☺️ 

I checked your [RSS feed here](https://simplepie.org/demo/?feed=https%3A%2F%2Fjhylin.github.io%2FData_in_life_blog%2Findex-r.xml) and it looks good to me ☺️ I also double-checked your feed for Python [here](https://simplepie.org/demo/?feed=https%3A%2F%2Fjhylin.github.io%2FData_in_life_blog%2Findex-python.xml) and it also looks the way it should be (and you're now all set for the PyLadies’ list as well 🤗)

I think what throws the remaining error is a missing comma ☺️ (I added a comment to the line where it needs to be)

image

File: blogs/jhylin.github.io.json (hunk 1 above)
Comment (jhylin) [reply blogs/jhylin.github.io.json]: Thank you @cosimameyer, I've just added the comma.

Comment (cosimameyer) [reply blogs/jhylin.github.io.json]: Good point, thanks so much! ☺️

**Review by drmowinckels — APPROVED** [review]

**Comment by drmowinckels** [comment]
/
Thank you for adding your blog to the list and for making free and open R content 😊 🥳 🎉

**Pull request merged**